src/components/header/renderer.tsx

```diff
@@ -134,7 +134,7 @@ class Header extends React.PureComponent<Props, State> {
 
     const HelpButton = (
       <div className="header-button help" onClick={() => this.setState(current => ({ ...current }))}>
```

Comment:
We already have the class `help` here. No need for another class.

Reply:
fixed!